helm/draft-annotations-api/templates/deployment.yaml

```diff
@@ -1,57 +1,57 @@
-
-apiVersion: extensions/v1beta1
-kind: Deployment
-metadata:
-  name: {{ .Values.service.name }} 
-  labels:
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}" 
-    visualize: "true" 
-    app: {{ .Values.service.name }} 
-spec:
-  replicas: {{ .Values.replicaCount }} 
-  selector: 
-    matchLabels:
-      app: {{ .Values.service.name }}
-  template:
-    metadata:
-      labels:
-        app: {{ .Values.service.name }}
-        visualize: "true" 
-    spec:
-      containers: 
-      - name: {{ .Values.service.name }} 
-        image: "{{ .Values.image.repository }}:{{ .Chart.Version }}"
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
-        env:
-        - name: ANNOTATIONS_RW_ENDPOINT
-          value: "http://generic-rw-aurora:8080"
-        - name: ANNOTATIONS_ENDPOINT 
-          valueFrom:
-            configMapKeyRef:
-              name: global-config
-              key: annotations-endpoint
-        - name: CONCEPT_SEARCH_ENDPOINT
-          valueFrom:
-            configMapKeyRef:
-              name: global-config
-              key: concept-search-endpoint
-        - name: UPP_APIKEY
-          valueFrom:
-            configMapKeyRef:
-              name: global-config
-              key: upp-api-key  
-        ports: 
-        - containerPort: 8080 
-        livenessProbe: 
-          tcpSocket: 
-            port: 8080 
-          initialDelaySeconds: 10 
-        readinessProbe: 
-          httpGet: 
-            path: "/__gtg" 
-            port: 8080 
-          initialDelaySeconds: 15 
-          periodSeconds: 30 
-        resources: 
-{{ toYaml .Values.resources | indent 12 }}
\ No newline at end of file
+##this is an example deployment.yaml that should be customized in order to meet the configuration for app k8s deployment
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.service.name }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    visualize: "true"
+    app: {{ .Values.service.name }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Values.service.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.service.name }}
+        visualize: "true"
+    spec:
+      containers:
+      - name: {{ .Values.service.name }}
+        image: "{{ .Values.image.repository }}:{{ .Chart.Version }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: ANNOTATIONS_RW_ENDPOINT
+          value: "http://generic-rw-aurora:8080"
+        - name: ANNOTATIONS_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: annotations-endpoint
+        - name: CONCEPT_SEARCH_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: concept-search-endpoint
+        - name: UPP_APIKEY
+          valueFrom:
+            secretKeyRef:
+              name: upp-api-key
```

Comment:
I think this is too generic a name if the namespace for secrets is across all services in the cluster.